packages/@lwc/babel-plugin-component/package.json

```diff
@@ -14,6 +14,9 @@
     "main": "src/index.js",
     "typings": "src/index.d.ts",
     "license": "MIT",
+    "scripts": {
+        "test": "jest"
+    },
```

Comment:
Adding this `test` command to every package is another downside of running `jest` in each package...

Reply:
After discussing with you and PM last week during plan, removed the jest tests as part of the nx targets.